.kilocode/workflows/README.md

```
@@ -0,0 +1,1288 @@
+2. Review code following code-review.md checklist
+3. Check for critical issues
+4. Fix any @1 priority items
+5. Run tests
+6. Stage changes
+7. Generate commit message following commit-conventions.md
+8. Commit
+9. Push
+10. Create PR with description
+
+**With Workflow:**
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)